node/consensus/engine.go

```diff
@@ -332,7 +341,7 @@ func (ce *ConsensusEngine) runConsensusEventLoop(ctx context.Context) error {
 	ce.log.Info("Starting the consensus event loop...")
 	catchUpTicker := time.NewTicker(5 * time.Second)
 	reannounceTicker := time.NewTicker(3 * time.Second)
-	blkPropTicker := time.NewTicker(1 * time.Second)
+	blkPropTicker := time.NewTicker(10 * time.Second)
```

Comment:
What's the significance of this duration value?  Should this be the configured propose timeout, or is this something else?

Reply:
blkPropTimer is what determines how quickly the node can catchup to the current consensus round (as this broadcasts the current block proposals), if this is pretty high, I observed that the block times are almost proportional to this.

Reply:
I will probably make this into a config